source/fundamentals/serialization.txt

```diff
@@ -140,6 +140,54 @@ Implementing the `IBsonArraySerializer
 interface enables the driver to access serialization information for individual
 items in an array.
 
+.. _csharp-convention-packs:
+
+Convention Packs
+----------------
+
+Convention packs allow you to define and apply
+**conventions** to your classes and their members during serialization.
+Conventions specify how data is mapped between your
+{+language+} objects and MongoDB documents without requiring you to decorate
+each class with attributes. 
+
+The {+driver-short+} provides built-in convention packs that you can use to
+customize the serialization process. The following table describes some of the
+built-in convention packs:
```

Comment:
The following are *conventions*, not *convention-packs*.

Convention-packs are sets of conventions that have been bundled to be used together.

Reply:
Thanks for the clarification. Updated the section to refer to conventions rather than convention packs in most instances